### PR TITLE
CNN Pooling

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-598/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

File: chapter_convolutional-neural-networks/pooling.md [hunk 1]
```diff
@@ -203,22 +204,23 @@ pool2d = tf.keras.layers.MaxPool2D(3, padding='same', strides=2)
...
-1. 池图层的计算成本是多少？假设池图层的输入大小为 $c\times h\times w$，则池窗口的形状为 $p_h\times p_w$，填充为 $(p_h, p_w)$，步幅为 $(s_h, s_w)$。
-1. 为什么您期望最大池和平均池的工作方式不同？
-1. 我们是否需要一个单独的最小池层？您可以用另一个操作替换它吗？
-1. 平均和最大池之间是否有另一个操作可以考虑（提示：召回 softmax）？为什么它可能不那么受欢迎？
+1. 你能将平均池化层作为卷积层的特殊情况实现吗？
+1. 你能将最大池化层作为卷积层的特殊情况实现吗？
+1. 假设池化层的输入大小为 $c\times h\times w$，则池化窗口的形状为 $p_h\times p_w$，填充为 $(p_h, p_w)$，步幅为 $(s_h, s_w)$。这个池化层的计算成本是多少？
+1. 为什么最大池化层和平均池化层的工作方式不同？
+1. 我们是否需要最小池化层？可以用已知函数替换它吗？
+1. 除了平均池化层和最大池化层，是否有其它函数可以考虑（提示：回忆 `softmax` ）？为什么它可能不受欢迎？
```
Comment (astonzhang) [inline chapter_convolutional-neural-networks/pooling.md:223]: 嗯，中文这样翻译也没问题。但英文那边between还是更好一点

**Comment by mli** [comment]
Job d2l-zh/PR-598/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

**Comment by mli** [comment]
Job d2l-zh/PR-598/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

**Comment by mli** [comment]
Job d2l-zh/PR-598/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

**Comment by mli** [comment]
Job d2l-zh/PR-598/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

**Comment by mli** [comment]
Job d2l-zh/PR-598/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-598/

**Pull request merged**